content/en/llm_observability/setup/sdk/python.md

```diff
@@ -637,6 +637,36 @@ def llm_call():
     return completion
 {{< /code-block >}}
 
+## Span processing
+
+To modify input and output data on spans a processor function can be configured. The processor function has access to span tags to enable conditional input/output modification. See the following examples for usage.
+
+### Example
+
+{{< code-block lang="python" >}}
+from ddtrace.llmobs import LLMObs
+from ddtrace.llmobs import LLMObsSpan
+
+def redact_processor(span: LLMObsSpan) -> LLMObsSpan:
+    if span.get_tag("no_output") == "true":
+        for message in span.output_messages:
+            message["content"] = ""
+    return span
+
+
+# If using LLMObs.enable()
+LLMObs.enable(
+  ...
+  span_processor=redact_processor,
+)
+# else when using `ddtrace-run`
+LLMObs.register_processor(redact_processor)
+
+with LLMObs.llm("invoke_llm_with_no_output"):
+    LLMObs.annotate(tags={"no_output": "true"})
```

Comment:
i wonder if we would also benefit from an example of using `LLMObs.annotation_context` here for auto-instrumented spans 🤔 or maybe linking to its section in the docs

Reply:
yes, great idea!